### PR TITLE
PMM-7557: Add backup bucket field

### PR DESCRIPTION
**Pull request by oter**
Task: https://jira.percona.com/browse/PMM-7557

API: https://github.com/percona/pmm/pull/691
pmm-managed https://github.com/percona/pmm-managed/pull/677

FB: https://github.com/Percona-Lab/pmm-submodules/pull/1546

### CODE REVIEW

File: management/backup/locations_test.go [hunk 1]
```diff
@@ -95,7 +96,6 @@ func TestAddWrongLocation(t *testing.T) {
 		pmmapitests.AssertAPIErrorf(t, err, 400, codes.InvalidArgument, "Missing location config.")
 		assert.Nil(t, resp)
 	})
-
```
Comment (artemgavrilov) [inline management/backup/locations_test.go:98]: I personally prefer to have these separate lines

**Review by artemgavrilov — APPROVED** [review]

File: management/backup/locations_test.go [hunk 2]
```diff
@@ -144,6 +144,25 @@
...
 	})
+	t.Run("missing s3 bucket", func(t *testing.T) {
+		t.Parallel()
+
+		resp, err := client.AddLocation(&locations.AddLocationParams{
+			Body: locations.AddLocationBody{
+				Name:        gofakeit.Name(),
+				Description: gofakeit.Question(),
+				S3Config: &locations.AddLocationParamsBodyS3Config{
+					Endpoint:   "http://example.com",
```
[the remark this answers is not in the document]
Comment (oter) [reply management/backup/locations_test.go]: fixed

[the remark this answers is not in the document]
Comment (oter) [reply management/backup/locations_test.go]: fixed

**Review by Dasio — APPROVED** [review]

File: management/backup/locations_test.go (hunk 1 above)
Comment (BupycHuk) [reply management/backup/locations_test.go:98]: +1

**Review by BupycHuk — APPROVED** [review]

**Pull request merged**